tests/integration/test_github.py

```diff
@@ -774,6 +774,17 @@ def test_user(self):
                 " encoded"
             )
 
+    def test_release_by_ghostuser(self):
+        """Test the ability to retrieve a release with "author: null" (ghost user)."""
+        cassette_name = self.cassette_name("release_author_null")
+        with self.recorder.use_cassette(cassette_name):
+            repository = self.gh.repository("qiskit-community", "qiskit-qec")
+        release = repository.release(63525446)
```

Comment:
```suggestion
            release = repository.release(63525446)
```

This makes an API call. This should be indented so that Betamax records it. You'll want to delete `tests/cassettes/GitHub_release_author_null.json` to re-create that file for simplicity.